src/Vehicle/Vehicle.cc

```diff
@@ -789,7 +789,7 @@ void Vehicle::_handleHilActuatorControls(mavlink_message_t &message)
 
 void Vehicle::_handleCommandAck(mavlink_message_t& message)
 {
-    bool showError = true;
+    bool showError = false;
```

Comment:
If we didn't explicitly register for an error to be shown (below) we should not show spurious messages to the user. Any such "debugging info" should go into the app log.